test/apps/iodemo/io_demo.cc

```diff
@@ -2607,6 +2613,8 @@ static int parse_args(int argc, char **argv, options_t *test_opts)
             std::cout << "  -D                          Enable debugging mode for IO operation timeouts" << std::endl;
             std::cout << "  -H                          Use human-readable timestamps" << std::endl;
             std::cout << "  -P <interval>               Set report printing interval"  << std::endl;
+            std::cout << "  -R <rndv-thresh>            Rendezvous threshold used to force eager" << std::endl;
```

Comment:
wired description
1) maybe "Rendezvous threshold used to switch to rendezvous protocol"?
2) what is defalt value? I think if not set it should be selected by UCX

Reply:
right, pls mention default behavior in description

Reply:
ok, updated description